src/ui/EngineRanking.ts

```diff
@@ -185,43 +185,45 @@ export default class EngineRanking extends ACellTableSection<RenderColumn> imple
 
     this.delayedUpdateAll = debounce(() => this.updateAll(), 50);
     this.delayedUpdateColumnWidths = debounce(() => this.updateColumnWidths(), 50);
-    ranking.on(`${Ranking.EVENT_ADD_COLUMN}.hist`, (col: Column, index: number) => {
-      this.columns.splice(index, 0, this.createCol(col, index));
-      this.reindex();
-      this.updateHist(col);
-      this.delayedUpdateAll();
-    });
-    ranking.on(`${Ranking.EVENT_REMOVE_COLUMN}.body`, (col: Column, index: number) => {
-      EngineRanking.disableListener(col);
-      this.columns.splice(index, 1);
-      this.reindex();
-      this.delayedUpdateAll();
-    });
-    ranking.on(`${Ranking.EVENT_MOVE_COLUMN}.body`, (col: Column, index: number, old: number) => {
-      //delete first
-      const c = this.columns.splice(old, 1)[0];
-      console.assert(c.c === col);
-      // adapt target index based on previous index, i.e shift by one
-      this.columns.splice(old < index ? index - 1 : index, 0, c);
-      this.reindex();
-      this.delayedUpdateAll();
-    });
-    ranking.on(`${Ranking.EVENT_COLUMN_VISIBILITY_CHANGED}.body`, (col: Column, _oldValue: boolean, newValue: boolean) => {
-      if (newValue) {
-        // become visible
-        const index = ranking.children.indexOf(col);
+    if (!noEvents) {
```

Comment:
Please add a comment why and when no event listner is added? Otherwise nobody understands why this flag exists.